src/box/xrow.c

```diff
@@ -423,6 +424,38 @@ iproto_reply_ok(struct obuf *out, uint64_t sync, uint32_t schema_version)
 	return 0;
 }
 
+int
+iproto_reply_id(struct obuf *out, uint64_t sync, uint32_t schema_version)
+{
+	size_t size = IPROTO_HEADER_LEN;
+	size += mp_sizeof_map(2);
+	size += mp_sizeof_uint(IPROTO_VERSION);
+	size += mp_sizeof_uint(IPROTO_CURRENT_VERSION);
+	size += mp_sizeof_uint(IPROTO_FEATURES);
+	size += mp_sizeof_iproto_features(&IPROTO_CURRENT_FEATURES);
+
+	char *buf = obuf_reserve(out, size);
```

Comment:
If the size is exact, you can use `obuf_alloc()` right away.

Reply:
This comment still awaits a response.

Reply:
[sorry, missed it - the comment is somehow invisible in the GitHub conversations view]

Sure, you're right. Fixed.